aeron-system-tests/src/test/java/io/aeron/SpySimulatedConnectionTest.java

```diff
@@ -276,6 +277,23 @@ else if (!isSubscriptionClosed)
         assertEquals(messagesToSend, fragmentCountSpy.value);
     }
 
+    @Test
+    @InterruptAfter(10)
+    void shouldNotHaveShortSendsWithMDCPublication()
+    {
+        launch();
+
+        final String channel = "aeron:udp?control=localhost:40456|control-mode=dynamic";
+
+        spy = client.addSubscription(spyForChannel(channel), STREAM_ID);
+        publication = client.addExclusivePublication(channel + "|ssc=true", STREAM_ID);
+
+        Tests.awaitConnected(spy);
+        Tests.awaitConnected(publication);
+
```

Comment:
Should call `waitUntilFullConnectivity()` which will also publish some data. Otherwise, how we are to check for short sends if nothing was sent yet?